app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoViewEffectHandler.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.introvideoscreen
+
+import org.simple.clinic.mobius.ViewEffectsHandler
+
+class IntroVideoViewEffectHandler(
+    private val uiActions: UiActions
```

Comment:
We don't need the uiactions yet. So ideally let's not add it in this commit. We should only add it as a parameter when we need it.

Reply:
We are migrating to use view effects so I think it's okay to have a few commits with implementations that are not being used yet. For example, here in this commit, we are not using the effect handler as well. This would also endure that in the next few commits while overriding effect handler we are passing the ui action.

Reply:
Should be fine